.github/workflows/docs.yml

```diff
@@ -26,16 +26,26 @@ jobs:
         with:
           python-version: 3.x
       - name: mkdocs install
-        run: pip install --upgrade pip && pip install -r docs/manual/requirements.txt
+        run: |
+          pip install --upgrade pip && pip install -r docs/manual/requirements.txt
       - name: git configuration
-        run: git config user.name 'github-actions[bot]' && git config user.email 'github-actions[bot]@users.noreply.github.com'
+        run: |
+          git config user.name 'github-actions[bot]' && git config user.email 'github-actions[bot]@users.noreply.github.com'
+      - name: check docs for warnings
+        working-directory: docs/manual
+        run: |
+          mkdocs build --no-directory-urls --strict
+      - name: check online help for warnings
+        working-directory: docs/manual
+        run: |
+          mkdocs build --no-directory-urls --strict --config-file help.yml
       - name: build docs without publishing them
-        if: ${{ github.event_name == 'pull_request' }}
+        if: github.event_name == 'pull_request' || github.ref != 'refs/heads/main'
         working-directory: docs/manual
         run: |
           mike deploy --title "4.4 Latest" --alias-type=copy --update-aliases 4.4 latest
-      - name: deploy latest docs to gh-pages branch
-        if: ${{ github.event_name != 'pull_request' }}
+      - name: deploy main docs to gh-pages 4.4 latest
+        if: github.event_name != 'pull_request' && github.ref == 'refs/heads/main'
```

Comment:
This line will need to be modified when backported, and when a major release is made.

I found that `if:` does not need the conditional ``${{`` syntax.